Sources/Apodini/Properties/Delegate.swift

```diff
@@ -153,7 +153,7 @@ extension Delegate {
 extension Delegate {
     /// Change a `delegate`'s `ObservedObject` to observe another `value`.
     @discardableResult
-    public func setObservable<V: ObservableObject>(_ keypath: WritableKeyPath<D, ObservedObject<V>>, to value: V) -> Delegate {
+    public func setObservable<V: ObservableObject>(_ keypath: KeyPath<D, ObservedObject<V>>, to value: V) -> Delegate {
```

Comment:
By migrating to requiring writable key paths anymore, you can remove the explicit setter in the AuthoirzedProperty of ApodiniAuthorization:

https://github.com/Apodini/Apodini/blob/9029e6899717fce389280f26e5c52a84f17a7444/Sources/ApodiniAuthorization/AuthorizedProperty.swift#L71-L73

Reply:
Removed in 54f20eec9d74639a56649865425b9b7b4666dce3 👍🏻